pkg/ui/key_test.go

```diff
@@ -51,31 +51,44 @@ var parseKeyTests = []struct {
 	{s: "F1", wantKey: K(F1)},
 
 	// Alt- keys are case-sensitive.
-	{s: "a-x", wantKey: Key{'x', Alt}},
-	{s: "a-X", wantKey: Key{'X', Alt}},
+	{s: "A-x", wantKey: Key{'x', Alt}},
+	{s: "A-X", wantKey: Key{'X', Alt}},
 
 	// Ctrl- keys are case-insensitive.
 	{s: "C-x", wantKey: Key{'X', Ctrl}},
 	{s: "C-X", wantKey: Key{'X', Ctrl}},
 
+	// Literal control chars are equivalent to the preferred Ctrl-? formulation.
```

Comment:
> Ctrl-?

I'd use "Ctrl-<key>" instead as this can be misunderstood as the Ctrl-? key combination.